XML/XML.sublime-syntax

```diff
@@ -13,11 +13,13 @@ file_extensions:
   - opml
   - svg
 first_line_match: |-
-    (?x)
-    ^(?:
-        <\?xml\s
-     |  \s*<([\w-]+):Envelope\s+xmlns:\1\s*=\s*"http://schemas.xmlsoap.org/soap/envelope/"\s*>
-     )
+    (?x:
+      ^(?:
+          <\?xml\s
+        | \s*<([\w-]+):Envelope\s+xmlns:\1\s*=\s*"http://schemas.xmlsoap.org/soap/envelope/"\s*>
+        | \s*<!DOCTYPE\s+(?!html\b)
```

Comment:
`\b` should probably be `(?=[ \t\n\f>])`. See https://html.spec.whatwg.org/multipage/parsing.html#doctype-name-state.

Reply:
Good catch.

Reply:
PS.: If I recall correctly, nested positive lookaheads don't work within other lookaheads.